ext/hal/nxp/mcux/boards/evkmimxrt1064/evkmimxrt1064_flexspi_nor_config.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2018 NXP
+ * All rights reserved.
```

Comment:
Import all the SDK one files in the same commit. The XIP/DCD commit will then just be CMake changes.

Reply:
done

Reply:
Please also update `ext/hal/nxp/mcux/README`